app/views/users/show.html.erb

```diff
@@ -1,28 +1,43 @@
-
 <!DOCTYPE html>
 <html>
 <head>
   <title>My Blog</title>
-  <link rel="stylesheet" type="text/css" href="styles.css">
+  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
 </head>
-<body>
-  <header>
-    <h1>Welcome to My Blog</h1>
-  </header>
-
-  <nav>
-    <ul>
-      <li><a href="/">Home</a></li>
-    </ul>
-  </nav>
-
-  <main>
-    <h2>User Profile</h2>
-    <!-- Add your user profile content here -->
-  </main>
 
-  <footer>
-    <p>&copy; 2022 My Blog. All rights reserved.</p>
-  </footer>
+<body id="body">
+  <header class="heading">
+    <h1><%= @user.name %> 's Profile</h1>
+    </header>
+    
+    <div class="userContainer">
+    <img src="<%= @user.photo %>" alt="user profile"/>
+  <div class="userCard">
+  <h2><%= @user.name %> </h2>
+  <p>Number of Posts: <%= @user.posts.count %></p>
+  </div>
+  </div>
+  
+  
+  <p class="bioSection">
+  <strong>Bio:</strong> 
+  <span><%= @user.bio %> <span>
+  </p>
+  
+  <% @recent_posts.each do |post| %>
+    <p class="bioSection">
+      <strong><%= post.title %></strong>
+      <span><%= post.text %></span>
+      <i>comments:  likes:</i>
```

Comment:
- **[OPTIONAL]** It is not possible to see the number of likes from the post, please 🙏🏽 fix it for a better UI.

![image](https://github.com/paulshaun-kdtk/blog-posts/assets/58642949/e38498d2-e943-488c-91a9-d3f242e7df1d)